classes/music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, archived, _genre, _author, _label)
```

Comment:
Kindly consider omitting `_genre`, `_author`, and `_label` parameters as they are not required in the MusicAlbum class.